src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -176,6 +213,7 @@ export default function PreferencesComponent() {
 					value={localAdminPref.defaultMeterMinimumDate}
 					onChange={e => makeLocalChanges('defaultMeterMinimumDate', e.target.value)}
 					placeholder='YYYY-MM-DD HH:MM:SS'
+					invalid={invalidDateLimits()}
```

Comment:
It might be nice to use a FormFeedback tag to give the user feedback on the invalid state. src/client/app/components/meters/CreateMeterModalComponent.tsx does this in several places.

Reply:
Thanks for the suggestion, I just added the FormFeedbacks, and their behavior should be the same as in CreateMeterModalComponent. Let me know if any other changes should be made.